src/commonMain/kotlin/exchange.dydx.abacus/state/v2/supervisor/AccountSupervisor.kt

```diff
@@ -751,12 +749,16 @@ internal open class AccountSupervisor(
         }
     }
 
-    internal open fun triggerCompliance(action: ComplianceAction, callback: TransactionCallback) {
+    internal open fun triggerCompliance(action: ComplianceAction, callback: TransactionCallback?) {
         if (compliance.status != ComplianceStatus.UNKNOWN) {
             updateCompliance(DydxAddress(accountAddress), compliance.status, action)
-            callback(true, null, null)
+            if (callback != null) {
+                callback(true, null, null)
+            }
+        }
+        if (callback != null) {
```

Comment:
Is this correct?  The callback gets called twice when `compliance.status != ComplianceStatus.UNKNOW`

Reply:
Oh good catch, fixing